src/bfx_tools/bwa.ml

```diff
@@ -319,8 +337,9 @@ let mem_align_to_bam
          ~sample_name:(Input_reads.sample_name input)
          ~read_group_id:(Input_reads.read_group_id input));
       "-t"; Int.to_string processors;
-      "-O"; Int.to_string configuration.Configuration.Mem.gap_open_penalty;
```

Comment:
Might be worth keeping this submodule in here (would make it easier to make mem vs. aln specific configs if needed). Probably doesn't matter, though…